solutions/import-workspace/locals_vpc.tf

```diff
@@ -0,0 +1,40 @@
+locals {
+  proxy_host_or_ip_port = join(":", [module.edge_vsi.vsi_details.ipv4_address, var.proxy_host.port])
+  nfs_host_or_ip_path   = join(":", [module.workload_vsi.vsi_details.ipv4_address, var.workload_host.nfs_path])
+}
+
+locals {
+  vsi_list  = [module.access_host.vsi_details, module.edge_vsi.vsi_details, module.workload_vsi.vsi_details]
+  vpc_names = [module.access_host.vsi_details.vpc_name, module.edge_vsi.vsi_details.vpc_name, module.workload_vsi.vsi_details.vpc_name]
+  vsi_names = [module.access_host.vsi_details.name, module.edge_vsi.vsi_details.name, module.workload_vsi.vsi_details.name]
+}
+
+locals {
+  cloud_connections = tolist([for each in data.ibm_tg_gateway.tgw_ds.connections : each.name if each.network_type == "directlink"])
+}
+
```

Comment:
Add block comments
####################################################
Locals for creating ACL rules
####################################################